lighthouse-core/audits/critical-request-chains.js

```diff
@@ -77,7 +129,10 @@ class CriticalRequestChains extends Audit {
         optimalValue: this.meta.optimalValue,
         extendedInfo: {
           formatter: Formatter.SUPPORTED_FORMATS.CRITICAL_REQUEST_CHAINS,
-          value: chains
+          value: {
+            chains,
+            longestChain: CriticalRequestChains._getLongestChain(chains)
```

Comment:
the change to the CriticalRequestChains `extendedInfo` is the only real externally-noticeable change. Added `longestChain` since it's used several times in the report, so no reason not to just precompute it